verification/io/router.gobra

```diff
@@ -23,33 +23,33 @@ package io
 // This function corresponds to function 'core' in the IO spec
 ghost
 decreases
-pure func core_as_set() set[IO_as]
+pure func (dp DataPlaneSpec) core_as_set() set[IO_as] //TODO: how
```

Comment:
I guess we can drop this function and just call `CoreAS`

Reply:
I would not drop this function. However, I would rename it to `core()`. This is the actual name in Isabelle. This way we would have a clear distinction between the IO-spec and the functions introduced by DataPlaneSpec.